packages/front-end/components/Experiment/ResultsTable.tsx

```diff
@@ -61,13 +64,46 @@ export default function ResultsTable({
   riskVariation,
   setRiskVariation,
   statsEngine,
+  pValueCorrection,
   sequentialTestingEnabled,
 }: ResultsTableProps) {
   const domain = useDomain(variations, rows);
   const orgSettings = useOrgSettings();
   statsEngine = statsEngine ?? orgSettings.statsEngine ?? "bayesian";
   sequentialTestingEnabled = sequentialTestingEnabled ?? false;
 
+  let pValueTooltipBody = null;
+  if (sequentialTestingEnabled) {
+    pValueTooltipBody = (
+      <div>
+        Sequential testing is enabled. These are &apos;always valid
+        p-values&apos; and robust to peeking. They have a slightly different
+        interpretation to normal p-values and can often be 1.000. Nonetheless,
+        the interpretation remains that the result is still statistically
+        significant if it drops below your threshold (
+        {orgSettings.pValueThreshold ?? 0.05}).
+      </div>
+    );
+  }
+  if (pValueCorrection) {
+    let correctionText = "all non-guardrail metrics and variations";
+    if (tableRowAxis === "dimension") {
+      correctionText =
+        "all dimension values, non-guardrail metrics, and variations";
+    }
+    pValueTooltipBody = (
+      <>
+        {pValueTooltipBody && <div className="mb-3">{pValueTooltipBody}</div>}
```

Comment:
this conditional wrapping/including of the old tooltip into the new one is a bit unorthodox. A more standard way would be something like...
```
let pValueTooltipBody = (<>
  ${sequentialTestingEnabled && (<div>Sequential stuff...</div>)}
  ${pValueCorrection && (<div>
    P-value stuff...
    ${tableAxies === "dimension" ? "all dimensional values..." : "all non-guardrail metrics..."}
    more p-value stuff...
  </div>)}
</>)

if (pValueCorrection) {
  // other changes not related to the tooltip
}
```
You could insert a bit more in-line logic for conditional mb-3 as well.